src/elm/Data/Increasable.elm

```diff
@@ -49,6 +50,11 @@ buildMultiplier =
     Multiplier
 
 
+identity : Multiplier
+identity =
```

Comment:
@johnschoeman `identity` in Elm already exists, as `identity : a -> a`, where it returns the value passed in, so I think this might be confusing coming back to it later.

@JoelQ In Haskell this would be `mempty` - would the Elm equivalent be `singleton` here?

Reply:
hmmm no `singleton` in Elm is usually equivalent to `pure` / ` return` in Haskell.

Perhaps just giving it a semantic name like `none`, or `noOp`?